newsletter-april-2023.rst

```diff
@@ -0,0 +1,84 @@
+.. post:: March 30, 2023
+   :tags: newsletter, python
+   :author: Ben
+   :location: MLM
+
+.. meta::
+   :description lang=en:
+      Company updates and new features from the last month,
+      current focus, and upcoming features.
+
+Read the Docs newsletter - April 2023
+=====================================
+
+News and updates
+----------------
+
+- 📚️ We refactored our user documentation to align with the `Diátaxis Framework <https://diataxis.fr>`__ and now the navigation sidebar and the landing page are so much better! 
+  Have a look at `docs.readthedocs.io <https://docs.readthedocs.io/en/stable/>`__.
+- 🌄️ The first proof-of-concept for a new API and JavaScript library is running and capable of displaying a menu matching our current :doc:`flyout menu <readthedocs:flyout-menu>`.
+  The big difference will be that the new API and library will be useful for any documentation framework or static site generator to allow for full control of the new flyout menu and access to API data.
+- ⚙️ Added a new build variable ``READTHEDOCS_CANONICAL_URL`` that's useful for projects that want to be aware of the canonical base URL while building.
+  `Read more in our docs <https://docs.readthedocs.io/en/stable/reference/environment-variables.html#envvar-READTHEDOCS_CANONICAL_URL>`__.
+- 📊️ All of our websites now use `Plausible <https://plausible.io/>`__ for analytics.
+- 🔒️ Fixed vulnerability: `Cache poisoning: serving arbitrary content on documentation sites  <https://github.com/readthedocs/readthedocs.org/security/advisories/GHSA-mp38-vprc-7hf5>`__
+
+You can always see the latest changes to our platforms in our :doc:`Read the Docs Changelog <readthedocs:changelog>`.
+
+.. figure:: img/screenshot-docs-diataxis-update.png
+   :alt: A screenshot of our current documentation after the refactor
+   
+   Here is how our documentation looks in April 2023.
+
+
+Upcoming features
+-----------------
+
+- 📚️ We are still doing changes in our documentation structure and content.
+- ⚡️ *A lot of work* is happening these days on bigger features.
+  In :doc:`the last newsletter </newsletter-march-2023>`, we mentioned the new Dashboard.
+  On the side of that,
+  we are also building a new generic JavaScript client and API that will give additional features to any documentation project or static site built on the platform.
```

Comment:
Shouldn't we split this into two?

- new theme (maybe with a screenshot as well?)
- the javascript client

Reply:
It's just that the new theme was already mentioned with a screenshot :) https://blog.readthedocs.com/newsletter-march-2023/

Reply:
Also added another notable mention from previous newsletter on about.readthedocs.com, I think it makes sense now as a separate item.